M3T1/BLL/CartService.cs

```diff
@@ -0,0 +1,28 @@
+﻿public class CartService : ICartService
+{
+    private readonly ICartRepository _repository;
+
+    public CartService(ICartRepository repository) => _repository = repository;
+
+    public Task<List<CartEntity>> GetAll() => _repository.GetByFilter();
```

Comment:
The requirements stay that we should get a list of the items of the Cart object. In your example i can see that it's returning all carts
<img width="444" alt="CleanShot 2023-03-24 at 12 13 28@2x" src="https://user-images.githubusercontent.com/15052879/227493637-70d2a6ee-a2f5-4e73-8168-37d64fdcae28.png">